04. Working with study/Curation script/README.md

```diff
@@ -45,21 +46,21 @@ You can test a curation rule before applying it with the --dry-run parameter. Th
 matches in the task output log (see later) but not actually change any data.
 
 ```bash
-odm-curate-study --rules <rules.json> --dry-run <study accession>
+odm-curate-study --rules <rules.json> --dry-run <study accession> -H GENESTACK_ENDPOINT_ADDR
 ```
 
 By default, if the target key already contains information it will not be overwritten and a warning will be noted in the
 logs, however you can force this using the --overwrite flag. This does not affect attributes set as 'read-only' in the
 template.
 
 ```bash
-odm-curate-study --overwrite <study accession>
+odm-curate-study --overwrite <study accession> -H GENESTACK_ENDPOINT_ADDR
```

Comment:
let's maybe add here --rules <rules.json>? cos anyway without mentioning rules file this example won't work